website/views/shared/tasks/task_view/graph.jade

```diff
@@ -1,6 +1,6 @@
 span.option-box.pull-right(ng-if='::main')
   a.option-action(ng-if='list.type=="todo"', ng-show='obj.history.todos', ng-click='toggleChart("todos")', tooltip=env.t('progress'), style='margin-right:5px;')
-    span.glyphicon.glyphicon-signal
+    span.glyphicon.glyphicon-signal(style={'zoom':1.5,'vertical-align':'-webkit-baseline-middle'})
```

Comment:
Why are there curly braces in this inline style? Also, we avoid inline styles, would prefer a class.